### ISSUE TITLE
Move to apollo-server repository

### ISSUE THREAD

**Issue by jeromemacias** [sirsavary/fastify-graphql#4]
Should be really cool to see https://github.com/apollographql/apollo-server/issues/626 fixed using this repo.

As you already done switch to TypeScript, it seems to be on the road of `apollo-server-fastify` ;)

Thanks for your works and @coopnd one !

**Comment by sirsavary** [comment]
There's a few small things that need to be done before I'd consider this ready for a move to `apollo-server`. I'll get things polished over the next few days and drop you a mention once things are good to go.

**Comment by jeromemacias** [comment]
Thanks! Can I help you ?

**Comment by sirsavary** [comment]
@jeromemacias I think I'll be alright to clean the code up myself. I'm going to borrow the `apollo-server-express` test suite and make sure that this middleware is compliant with it. Once that's done I'll ping you and we can start migrating it into `apollo-server`.

**Comment by sirsavary** [comment]
@jeromemacias Just thought I'd let you know that AFAIK this plugin should now be on par with the other Apollo middlewares as of 42f18e846d23fdb8b87757b49cec80abc6c55c2e

**Comment by ghost** [comment]
Thanks for helping out my lazy ass.  This has been a long time coming....

**Comment by tomByrer** [comment]
Seems someone else is [working on it](https://github.com/apollographql/apollo-server/issues/626#ref-commit-55f2975) the past 2 weeks if you 2 want to help?
